packages/embark/src/lib/core/ipc.js

```diff
@@ -118,6 +118,10 @@ class IPC {
       cb = cb || (() => {});
       return cb();
     }
+    this.queueRequest(action, data, cb);
```

Comment:
We don't really need this anymore do we? 😄

Reply:
Good point. Fixing.